### DISCUSSION TITLE
推荐应用这里有点问题，已经安装的应用安装按钮是否可点击不一致

### DISCUSSION THREAD

**Discussion by IshuaWang** [1Panel-dev/1Panel#7290] [General]
如下图，两个应用其实都安装了，但是第一个按钮无法点击，第二个可以点击，这就有点奇怪。
![image](https://github.com/user-attachments/assets/ff4b6e6d-2ddd-4a44-8224-0a05d91d9b16)

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们计划在 v2 版本中优化推荐应用的逻辑。
